Bottom: ea982dcbee5936daeda6492810167e6e420e16a9
Top:    c1d2a82e96998a0a95ebfe5c64bb294609dc4807
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-10 22:56:18 -0500

Refresh of fix-outline-drawing-to-only-ap

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 8927fa4..bbc5524 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1000,8 +1000,8 @@ ps_set_layer (const char *name, int group, int empty)
       global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
-      strcmp (name, "outline") &&
-      strcmp (name, "route")
+      strcmp (name, "outline") != 0 &&
+      strcmp (name, "route") != 0
       )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
